apps/backend/charity/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { connectDB, setupTracing } from "../../lib/src/otel";
+setupTracing("charity");
+
+import express, { Request, Response, NextFunction } from "express";
+import morgan from "morgan";
+import cookieParser from "cookie-parser";
+import { AuthenticatedRequest, authenticate_request } from "../../lib/src/auth";
+import { createCharity } from "./createCharity";
+import { getCharities } from "./getCharities";
+import { getCurrentCharity } from "./getCurrentCharity";
+import { deleteCharity } from "./deleteCharity";
+
+const PORT = 8215;
+
+const app = express();
+
+app.use(morgan("dev"));
```

Comment:
```suggestion
app.use(
  morgan("dev", {
    skip: function (req: Request, res: Response) {
      return res.statusCode < 400;
    },
  }),
);
```
As per the other backend services